modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/EpicsUtil.scala

```diff
@@ -65,9 +65,8 @@ trait EpicsSystem[T] {
 }
 
 object EpicsCommand {
-  def safe[A](a: SeqAction[A]): SeqAction[A] = EitherT(a.run.attempt.map {
-    case -\/(t) => SeqexecException(t).left
-    case \/-(e) => e
+  def safe[A](a: SeqAction[A]): SeqAction[A] = EitherT(a.run.handle {
+    case e: Exception => SeqexecException(e).left
```

Comment:
Yes. Strictly speaking, Throwables of the subclass Exception will be caught and passed as a (bad) result, while Throwables of the subclass Error will cause the application to terminate. That group includes errors like OutOfMemory and StackOverflow.